### DISCUSSION TITLE
Will different versions of kernel affect performance?

### DISCUSSION THREAD

**Discussion by Lewisyixin** [apache/apisix#6795] [Q&A]
Default kernel we use is just 3.10 in Centos7.3, which are a little old. 
I just wan't to know if this  will affect performance of apisix?

**Comment by tokers** [comment]
Maybe, since different kernel might have different default parameters settings, optimizations (like the TCP stack).

**Reply by Lewisyixin** [reply]
Is there any advice to provide? Like kernel verions >= x.x.x?

**Reply by tokers** [reply]
Nope, all Linux releases supported by OpenResty can also be supported by APISIX.
